### PR TITLE
Adding a page on firebase.

### PR DESCRIPTION
**Pull request by sfshaza2**
Not sure who owns Firebase these days. @wmleler, @filiph, please review!  (This is on the new website.)

### CODE REVIEW

**Comment by filiph** [comment]
LGTM

cc @kroikie

File: src/_data/sidenav.yml [hunk 1]
```diff
@@ -83,6 +83,7 @@
           permalink: /docs/development/data-and-backend/json
         - title: Back-end services
         - title: Use Firebase with Flutter
```
Comment (chalin) [inline src/_data/sidenav.yml]: The title should match the page title, namely just "Firebase".

File: src/docs/development/data-and-backend/firebase.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Firebase
+description: Docs on how to use Firebase and Firestore with Flutter.
```
Comment (chalin) [inline src/docs/development/data-and-backend/firebase.md]: Docs on how ... -> How to ...

File: src/docs/development/data-and-backend/firebase.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+Firebase is a Backend-as-a-Service (BaaS) app development platform
+that you can use as a realtime database, for file storage, for
+authentication, and as a hosting service for your static files.
+
+And Firebase supports Flutter.
+
+For more information, see:
```
Comment (chalin) [inline src/docs/development/data-and-backend/firebase.md]: Maybe:

> Firebase works seamlessly with Flutter. For more information, see:

**Comment by sfshaza2** [comment]
Gonna run with Filip's LGTM.

File: src/docs/development/data-and-backend/firebase.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Firebase works seamlessly with Flutter. For more information, see:
+
+* [Getting started with Firebase and
+   Flutter](https://firebase.google.com/docs/flutter/setup)
+   in the [Firebase docs](https://firebase.google.com/docs)
+* [Firebase for Flutter
+   codelab](https://codelabs.developers.google.com/codelabs/flutter-firebase/#0)
+* [Using Firestore as a backend to your Flutter
+   app](https://youtu.be/DqJ_KjFzL9I) (video)
+
```
Comment (wmleler) [inline src/docs/development/data-and-backend/firebase.md]: Please add the FlutterFire plugins page
https://github.com/flutter/plugins/blob/master/FlutterFire.md

File: src/docs/development/data-and-backend/firebase.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+title: Firebase
+description: How to use Firebase and Firestore with Flutter.
+---
+
+Firebase is a Backend-as-a-Service (BaaS) app development platform
```
[the remark this answers is not in the document]
Comment (sfshaza2) [reply src/docs/development/data-and-backend/firebase.md]: Nice!

**Comment by sfshaza2** [comment]
Added Wm's feedback in another PR.

**Pull request merged**